packages/web-components/src/button/button.styles.ts

```diff
@@ -0,0 +1,336 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusLarge,
+  borderRadiusMedium,
+  borderRadiusNone,
+  borderRadiusSmall,
+  colorBrandBackground,
+  colorBrandBackgroundHover,
+  colorBrandBackgroundPressed,
+  colorNeutralBackground1,
+  colorNeutralBackground1Hover,
+  colorNeutralBackground1Pressed,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForeground1,
+  colorNeutralForeground1Hover,
+  colorNeutralForeground1Pressed,
+  colorNeutralForeground2,
+  colorNeutralForeground2BrandHover,
+  colorNeutralForeground2BrandPressed,
+  colorNeutralForeground2Hover,
+  colorNeutralForeground2Pressed,
+  colorNeutralForegroundDisabled,
+  colorNeutralForegroundOnBrand,
+  colorNeutralStroke1,
+  colorNeutralStroke1Hover,
+  colorNeutralStroke1Pressed,
+  colorNeutralStrokeDisabled,
+  colorStrokeFocus2,
+  colorSubtleBackground,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  colorTransparentBackground,
+  colorTransparentBackgroundHover,
+  colorTransparentBackgroundPressed,
+  colorTransparentStroke,
+  curveEasyEase,
+  durationFaster,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  fontWeightSemibold,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+  shadow2,
+  shadow4,
+  spacingHorizontalL,
+  spacingHorizontalM,
+  spacingHorizontalS,
+  spacingHorizontalSNudge,
+  spacingHorizontalXS,
+  strokeWidthThick,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+const buttonSpacingSmall = '3px';
+const buttonSpacingSmallWithIcon = '1px';
+const buttonSpacingMedium = '5px';
+const buttonSpacingLarge = '8px';
+const buttonSpacingLargeWithIcon = '7px';
+
+// Need to support icon hover styles
+export const styles = css`
```

Comment:
Its my assumption we will abstract these if we share anything with `fluent-anchor`?

Reply:
Essentially - If we can do it with one monolithic stylesheet I plan on attempting that first though as everything would be "shared" in terms of visual API.